webapp/components/ContributionForm/index.vue

```diff
@@ -106,15 +117,25 @@ export default {
   },
   methods: {
     submit() {
+      const { title, content, teaserImage } = this.form
+      let language
+      if (this.form.language) {
+        language = this.form.language.value
+      } else if (this.contribution && this.contribution.language) {
+        language = this.contribution.language
+      } else {
+        language = this.$i18n.locale()
+      }
       this.loading = true
       this.$apollo
         .mutate({
           mutation: this.id ? PostMutations().UpdatePost : PostMutations().CreatePost,
           variables: {
             id: this.id,
-            title: this.form.title,
-            content: this.form.content,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-24T17:51:46Z" title="Monday, June 24th 2019, 7:51:46 pm +02:00">Jun 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -75,6 +107,13 @@ export default {
       disabled: false,
       slug: null,
       users: [],
+      dropzoneOptions: {
+        url: 'https://httpbin.org/post',
```


As @Tirokk noted, this is a temporary solution. We should prioritize adding our own restful endpoint for the images to be uploaded to, and then store them in the backend upon form submit.

If there is a better temporary solution, I can change this